spec/services/process_submission_service_spec.rb

```diff
@@ -190,7 +190,7 @@
       end
 
       it 'dispatches 1 email for each submission email attachment' do
-        expect(EmailService).to receive(:send_mail).exactly(5).times
+        expect(EmailService).to receive(:send_mail).exactly(4).times
```

Comment:
why are there fewer emails sent?

Reply:
The test setup has changed.
I looked at the detail objects and there are only 4 to act upon.